docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Sphinx is great but I was hoping to make https://docs.cacophony.org.nz/ the place to host all our documentation.

That wouldn't address versioning as nicely as something like https://readthedocs.org/ though... Perhaps we host the Python API docs at https://readthedocs.org/ (generated from Sphinx here) and then have an article on https://docs.cacophony.org.nz/ which links to the Python and Go API docs. How does that sound to you?

Reply:
Yes this Sphinx Docs is  only the skeleton so far as I was not sure how to self document to  https://docs.cacophony.org.nz/ , I like your suggestion of a link from https://docs.cacophony.org.nz/  to Python API docs at https://readthedocs.org/  no extra workflow and is consistent.

Reply:
@mjs raised #3, marking this as resolved